src/Context/QuizContext.jsx

```diff
@@ -5,7 +5,52 @@ const QuizContext = createContext();
 const QuizProvider = ({ children }) => {
   const [state, dispatch] = useReducer(QuizReducer, {
```

Comment:
more specific name can be used here for state and dispatch

Reply:
Okay, I Will change.